src/content/docs/apm/agents/php-agent/other-integrations/magento-integration.mdx

```diff
@@ -0,0 +1,291 @@
+---
+title: Magento monitoring integration
+tags:
+    - New Relic integrations
+    - Magento
+metaDescription: Monitor your Magento application.
+---
+
+import apmMagentoDashboardOverview from 'images/apm_screenshot-full_magento-dashboard-overview.png'
+
+Our Magento integration monitors the performance of your Magento ecommerce application, helping you diagnose issues in your app's performance and respond by optimizing your code. Our Magento integration relies on these New Relic agents and tool:
+
+* PHP agent
+* MySQL agent
+* Apache agent
+* Elasticsearch agent
+* (Optional) Infrastructure agent
+* NRI-flex (Our all-in-one tool that allows you to collect metric data from a wide variety of services.)
```

Comment:
Is NR Flex also optional since it comes bundled with the infra agent (and we say the infra agent is optional)?